src/bundle/Resources/public/css/modules/modal.css

```diff
@@ -0,0 +1,31 @@
+.ez-modal {
+    position: fixed;
+    padding: 0;
+    border: 0;
+    border-radius: 8px;
+    background: var(--ez-color-ground-modal);
+
+    box-shadow: 0 3px 20px var(--ez-color-shadow-modal);
+}
+
+.ez-modal header, .ez-modal section, .ez-modal footer {
+    padding: 1em;
+}
+
+.ez-modal header {
```

Comment:
The `confirm-dialog` component does not have the `header` part.
Do we need a follow-up issue for this?